api/Window.json

```diff
@@ -4525,11 +4525,11 @@
             },
             "firefox": {
               "version_added": "1",
-              "notes": "From Firefox 82, if a new page from another domain is loaded into a tab, then <code>window.name</code> is set to the empty string (the original string is restored if the tab reverts back to the original page). This change prevents some cross-site attacks. See <a href='https://bugzil.la/444222'>bug 444222</a>."
+              "notes": "From Firefox 86, if a new page from another domain is loaded into a tab, then window.name is set to an empty string (the original string is restored if the tab reverts back to the original page). This change prevents some cross-site attacks. From Firefox 82 to 86 this feature can enabled using the preference privacy.window.name.update.enabled. See <a href='https://bugzil.la/1685089'>bug 1685089</a> and <a href='https://bugzil.la/444222'>bug 444222</a>."
```

Comment:
```suggestion
              "notes": "Before Firefox 86, if a new page from another domain is loaded into a tab, then <code>window.name</code> is not set to an empty string, which can allow some cross-site attacks. See <a href='https://bugzil.la/1685089'>bug 1685089</a> and <a href='https://bugzil.la/444222'>bug 444222</a>."
```

Reply:
This looks great to me @ddbeck and please don't be sorry! I feel I learn something new every time I do a BCD PR ☺️ and am very grateful for your continued feedback!

I wholeheartedly agree with this approach, esp considering Chrome are very likely to push this soon too.

Also I assume then that we don't need to mention it being behind a pref from 82 - 86?

Reply:
> Also I assume then that we don't need to mention it being behind a pref from 82 - 86?

Yes, I think so. If it hadn't shipped yet and the shipping version was indeterminate, I might suggest a mentioning it. But it'll be a historical footnote starting tomorrow (which will be before this data even gets into a BCD release).